ros2/transforms/launch/tf_launch.launch.py

```diff
@@ -5,11 +5,15 @@
 # This function is always needed
 def generate_launch_description():
 
-    # Declare a variable Node for each node
-    robot_state_publisher = Node(
-        package="robot_state_publisher",
-        executable="robot_state_publisher",
-        arguments=["/home/jose/ws/ros2/foxglove_ws/src/ros2_tutorials/tf_pkg/urdf/robot.urdf"]
+    arm_frame_publisher = Node(
```

Comment:
It seems like indentation here is still 4 spaces.

Is it possible to do 2, to keep consistent (like your JSON file above)?